app/build.gradle.kts

```diff
@@ -87,4 +87,6 @@ dependencies {
     implementation(project(":feature:detail"))
     implementation(project(":feature:recipe"))
     implementation(project(":feature:collection"))
+
+    implementation("io.airbridge:sdk-android:4.1.0")
```

Comment:
Added Airbridge SDK dependency. This is necessary to integrate Airbridge functionality into the app. The Airbridge SDK provides features like deep linking, attribution, and analytics. Make sure to replace '4.1.0' with the latest version of the SDK. As per the Airbridge SDK documentation, this dependency is required for the SDK to function properly.